src/mam/mod_mam_muc_rdbms_arch.erl

```diff
@@ -334,6 +334,23 @@ lookup_messages(Host, RoomID, RoomJID = #jid{},
     {ok, {TotalCount, Offset,
           rows_to_uniform_format(MessageRows, Host, RoomJID)}}.
 
+-spec get_mam_muc_gdpr_data(jid:username(), jid:server()) ->
+    {ok, ejabberd_gen_mam_archive:mam_muc_gdpr_data()}.
+get_mam_muc_gdpr_data(Username, Host) ->
+    LUser = jid:nodeprep(Username),
+    LServer = jid:nodeprep(Host),
+    WithJID = jid:to_binary({LUser, LServer}),
+    %% TODO What is PageSize ?
```

Comment:
PageSize is usually 50.
How many messages to return at once.
Not needed to worry about here, we need them all.